Plugin.php

```diff
@@ -34,6 +43,7 @@ public function registerNavigation()
                 'url'      => Backend::url('october/test/people'),
                 'icon'     => 'icon-child',
                 'order'    => 200,
+                'permissions' => ['october.test.access'],
```

Comment:
Permissions also need to be checked in all controllers.

Reply:
@mjauvin I didn't mean add the permission to the side menus, that's unnecessary when it's been added to the main menu. I meant add it to the Controller files themselves, otherwise people can still access the controllers by just navigating directly to the URL.